fk/secretsend.go

```diff
@@ -104,6 +106,42 @@ https://download.fluidkeys.com#` + recipientEmail + `
 	return 0
 }
 
+func getSecretFromFile(filename string) (*string, error) {
+	secretData, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, fmt.Errorf("error reading file: " + err.Error())
+	}
+	secret := string(secretData)
+	out.Print("---\n")
+	out.Print(secret)
+	out.Print("---\n\n")
+
+	prompter := interactiveYesNoPrompter{}
+
+	if prompter.promptYesNo("Send "+filename+"?", "y", nil) {
```

Comment:
This is the cause of the crash. If you answer "no" it just returns a `nil` string pointer and *nil error*.

I didn't spot this from view code because *normally* we use the final return as a *successful* return, and previous returns as failure cases.

I'd strongly suggest flipping the logic here to test is `promptYesNo` is `false`, and use the final return as the success argument.

(and as per my other comment, return a `string` not a `*string`)

Reply:
Thanks!